gtsam/hybrid/HybridConditional.cpp

```diff
@@ -129,6 +129,22 @@ double HybridConditional::error(const HybridValues &values) const {
       "HybridConditional::error: conditional type not handled");
 }
 
+/* ************************************************************************ */
+AlgebraicDecisionTree<Key> HybridConditional::errorTree(
```

Comment:
Why not in HybridFactor?

Reply:
`HybridFactor` doesn't have the `asGaussian`, `asHybrid` and `asDiscrete` methods.